src/tests/Common/wasm-test-runner/WasmTestRunner.proj

```diff
@@ -10,6 +10,7 @@
     <NETCoreAppMaximumVersion>99.0</NETCoreAppMaximumVersion>
 
     <WasmAppBuilderTasksAssemblyPath>$(CORE_ROOT)\WasmAppBuilder\WasmAppBuilder.dll</WasmAppBuilderTasksAssemblyPath>
+    <MonoAOTCompilerTasksAssemblyPath>$(CORE_ROOT)\MonoAOTCompiler\MonoAOTCompiler.dll</MonoAOTCompilerTasksAssemblyPath>
```

Comment:
So, we haven't actually made the changes necessary for the runtime tests to use AOT, right? Is this property used by the libraries tests?

Reply:
In theory, if you set the right inputs, the runtime tests could take advantage.  I didn't set that up... Only the libraries tests.

Reply:
I guess my question was; does it belong in this PR? Because right now nothing is using it.

Reply:
I think we add enabling it for the runtime tests later.